src/ui/components/ThemeImage/styles.scss

```diff
@@ -0,0 +1,30 @@
+@import '~core/css/inc/mixins';
+@import '~amo/css/inc/vars';
+@import '~ui/css/vars';
+
+.ThemeImage {
+  height: auto;
+  max-height: $theme-height-legacy;
+  max-width: $theme-width-default;
+  order: -1;
+  overflow-y: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.ThemeImage-image {
+  display: block;
+  height: auto;
+  max-width: $theme-width-default;
+  object-fit: cover;
+  object-position: top left;
```

Comment:
Can we really remove those lines?

Reply:
I really think it's fine to remove now since there is no more cropping and it it should always be 100% width but we can probably circle back since it was already like that now and doesn't mess anything up

Reply:
I am fine with removing those lines, but maybe after the tag? :D